docs/components/radio.md

```diff
@@ -68,9 +72,59 @@ Will place description text under the radio label (required). Can also be [slott
 
 <KRadio :selected-value="true" v-model="radio" label="Label Example" description="Some subheader text" />
 
-### selectedValue
+### type
 
-The value of the `KRadio` option that will be emitted by the `change` and `update:modelValue` events.
+Controlls apperance of radio input element. Accpted values: `default`(default) and `card`.
+
+<KCard>
+  <template v-slot:body>
+    <div class="d-flex">
+      <KRadio type="card" label="Foo" description="This subheader" selected-value="foo" v-model="cardRadio">
+        <template v-slot:before>
+          <img src="/img/kong-logomark.png" alt="Kong logo" />
+        </template>
+      </KRadio>
+      <KRadio type="card" label="Bar" description="That subheader" selected-value="bar" v-model="cardRadio">
+        <template v-slot:before>
+          <img src="/img/kong-logomark.png" style="transform: rotate(180deg);" alt="Kong logo" />
+        </template>
+      </KRadio>
+    </div>
+    <div class="mt-3">Selected: {{ cardRadio }}</div>
+  </template>
+</KCard>
+
+```html
+<template>
+  <KRadio type="card" label="Foo" description="This subheader" selected-value="foo" v-model="cardRadio">
+    <template v-slot:before>
+      🦍
+    </template>
+  </KRadio>
+  <KRadio type="card" label="Bar" description="That subheader" selected-value="barv-model="cardRadio">
+    <template v-slot:before>
+      🦍
+    </template>
+  </KRadio>
```

Comment:
Unsure why this second radio is showing the icon upside-down?

![image](https://user-images.githubusercontent.com/2229946/214116768-f183cca7-93a4-4730-a13c-5e5d0c48d1ef.png)